src/xdocs/property_types.xml

```diff
@@ -30,21 +30,28 @@
     </p>
     </section>
 
-    <section name="integer">
+    <section name="Integer">
       <p>
         This type represents an integer. The string representation is
         parsed using the <code>java.lang.Integer</code> class.
       </p>
     </section>
 
-    <section name="string">
+    <section name="Double">
+      <p>
+        This type represents an integer. The string representation is
+        parsed using the <code>java.lang.Double</code> class.
+      </p>
+    </section>
+
+    <section name="String">
       <p>
         This type represents a string. The literal string representation
         is used.
       </p>
     </section>
 
-    <section name="boolean">
+    <section name="Boolean">
       <p>
         This type represents a boolean. The default value is <code>false</code>.
```

Comment:
@romani Isn't `The default value` misleading? Default value is determined by check, and we don't always default to `false`. I thought this was more for the string representation, but we have a statement later covering that.

Reply:
yes, good catch.
@gaurabdg , please remove this line in separate commit or PR.

Reply:
Done.